src/Mutable.php

```diff
@@ -146,7 +146,7 @@ public function generate()
             }
 
             //TODO: handle whitespace!
-            if (is_array($token) && $token[0] == T_FUNCTION) {
+            if (is_array($token) && $token[0] == T_FUNCTION && $className !== '???') {
```

Comment:
To avoid things like `$token[0] = T_FUNCTION`. We could argue about having assignments in a function call or if condition, but it's sometimes used. Yoda condition in those cases prevent those mistakes